### DISCUSSION TITLE
[Valheim] Could not find method due to a type load error

### DISCUSSION THREAD

**Discussion by Bodzio0** [GameServerManagers/LinuxGSM#3548] [Q&A] [answered]
Hi there! 
I have some issues with running the Valheim server on Ubuntu 20.04.3 LTS with the 5.4.0-81-generic kernel. In the console is a lot of error with unity (i think, logs in attach). 
The server is running in my local network in ESXi with free 90% CPU, 6GB RAM, and 120GB HDD. 

I've run 2 different game servers on that VM (Minecraft, terraria) without any issue. I've tried to build a server on a new user but with the same results :( 

If u have any suggestions I will be grateful :) 
[vhserver-console-2021-09-01-21_14_46.log](https://github.com/GameServerManagers/LinuxGSM/files/7109849/vhserver-console-2021-09-01-21_14_46.log)

**Comment by johnoclockdk** [comment]
try to do ./vhserver validate and try again

**Reply by Bodzio0** [reply]
Thanks for you answer. Unfortunately it didn’t work :(
In validate logs everything is legit…

**Comment by Bodzio0** [comment, marked as answer]
Okay, I have a solution. U need to run server on vhserver as user name
